.github/workflows/publish.yaml

```diff
@@ -0,0 +1,17 @@
+name: publish
+on:
+  push:
+    branches: [master, main]
+jobs:
+  tag:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@v2
+      - uses: Saionaro/extract-package-version@v1.1.1
```

Comment:
Does this tag with the version from `package.json`? What if you make a commit and don't change the version number?

Reply:
This reads the package version, with the output subsequently used as the tag version via its `custom_tag` parameter; it's up to the coder and request reviewer to ensure the major.minor.[patch|revision] is bumped appropriately. I'm open to suggestions regarding bump/patch implementation; [this looks interesting](https://github.com/mathieudutour/github-tag-action#bumping).

Reply:
Skipping the tag creation would be acceptable behavior:

* ensures the version is an important part of a PR (no bump, no release!)
* new branch and PR must be created for the bump'n'tag